build_runner/test/common/package_graphs.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build_runner/build_runner.dart';
+
+PackageGraph buildGraph(
```

Comment:
nit: buildPackageGraph?

Reply:
Was going for terseness but I suppose it could be confused with AssetGraph...